### PR TITLE
left align text for quotes on customers and pricing page; footer link text change

### PR DESCRIPTION
**Pull request by mgyorffy**
Customers page:
Screen Shot 2021-04-03 at 1 08 46 PM

Pricing page:
Screen Shot 2021-04-03 at 1 40 13 PM
Screen Shot 2021-04-03 at 1 40 07 PM

Updated footer link:
Screen Shot 2021-04-03 at 2 05 46 PM

@attfarhan @marekweb @tommypez

### CODE REVIEW

File: website/src/pages/customers.tsx [hunk 1]
```diff
@@ -241,7 +241,7 @@ export default ((props: any) => (
                         <p>Watch how Sid Sijbrandij (GitLab CEO) and Quinn Slack (Sourcegraph CEO) do code reviews.</p>
                     </div>
                     <div className="row justify-content-center pt-md-4">
-                        <div className="col-lg-10 text-center">
+                        <div className="col-lg-10">
```
Comment (attfarhan) [inline website/src/pages/customers.tsx:244]: After this change, the logo for Workiva is left aligned, whereas all the other logos are still centered. I think there's a fix needed where the `img` tag needs to be [wrapped in a div, like the other logos](https://github.com/sourcegraph/about/pull/2872/files#diff-ad3c9221af8145935b3f59e56855ef1ea64b6af665c8552afb63d8c42acf8669R314)

File: website/src/components/Footer.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const Footer: React.FunctionComponent<{ minimal?: boolean }> = ({ minimal
                                         rel="noopener"
                                         className="bigCode"
                                     >
-                                        Not a developer?
+                                        Sourcegraph Overview (PDF)
```
Comment (attfarhan) [inline website/src/components/Footer.tsx:33]:
```suggestion
                                        Sourcegraph overview (PDF)
```

Using sentence case as per the guidelines in https://about.sourcegraph.com/handbook/communication/content_guidelines/actionable_language#headings-and-subheadings.

**Review by attfarhan — CHANGES_REQUESTED** [review]
Left a couple comments. Code looks good to me otherwise. However, was the left-aligning of quotes a request from someone? How can I find out where these action items come from? 

I saw the Not a developer -> Sourcegraph overview in Asana (https://app.asana.com/0/1200095757619647/1200095757619650), but didn't see anything for left aligning quotes.

File: website/src/components/Footer.tsx (hunk 2 above)
Comment (mgyorffy) [reply website/src/components/Footer.tsx:33]: fixed

File: website/src/pages/customers.tsx (hunk 1 above)
Comment (mgyorffy) [reply website/src/pages/customers.tsx:244]: good catch. thank you. updated it.

**Review by attfarhan — APPROVED** [review]

File: website/src/pages/customers.tsx (hunk 1 above)
Comment (tommypez) [reply website/src/pages/customers.tsx:244]: Thanks guys. This request was from me, we started an Asana board to track our requests to Mate for the marketing site.

**Pull request merged**